DistroLauncher/SetOnceNamedEvent.cpp

```diff
@@ -26,6 +26,7 @@ namespace Win32Utils
         HANDLE handle = CreateEvent(nullptr, TRUE, FALSE, name);
         if (handle != nullptr) {
             event = handle;
+            setOnce = false;
         }
```

Comment:
This `if` block is unnecessary, the alternative does the same.

You could assign the event to the handle directly.

Reply:
Agreed